### PR TITLE
Feature/send user messages

### PR DESCRIPTION
**Pull request by nburkley**
Adds module to send messages to user using slack Web API.

Sends user an initial greeting message when the OAuth authentication is complete.

### CODE REVIEW

**Comment by nburkley** [comment]
ping @edenspiekermann/backend

File: priv/gettext/default.pot [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Run `mix gettext.extract` to bring this file up to
+## date. Leave `msgstr`s empty as changing them here as no
+## effect: edit them in PO (`.po`) files instead.
+msgid ""
+msgstr ""
+"Language: en\n"
+
+#: web/templates/page/index.html.eex:2
+msgid "Welcome to %{name}"
+msgstr "somethign somethign to %{name}"
```
Comment (ghost) [inline priv/gettext/default.pot]: I know it’s only a placeholder, but there is a typo in `something`.

File: web/models/auth_handler.ex [hunk 2]
```diff
@@ -6,10 +6,12 @@ defmodule EspiDni.AuthHandler do
...
+  import EspiDni.Gettext
 
   def init_from_auth(%Auth{} = auth) do
     with {:ok, team} <- TeamFromAuth.find_or_create(auth),
-         {:ok, user} <- UserFromAuth.find_or_create(auth, team) do
+         {:ok, user} <- UserFromAuth.find_or_create(auth, team),
+         {:ok, user} <- send_welcome_message(user) do
       {:ok, team, user}
     end
     |> case do
```
Comment (ghost) [inline web/models/auth_handler.ex:17]: Is this a common way to do the `when`? Looks a bit hard to read what's going on. Could the `case` part be moved into a private function?

File: lib/espi_dni/slack_web.ex [hunk 3]
```diff
@@ -0,0 +1,24 @@
+defmodule EspiDni.SlackWeb do
+  use Slack
+
+  import Ecto.Query, only: [from: 1, from: 2]
```
Comment (ghost) [inline lib/espi_dni/slack_web.ex:4]: 👍  It’s really cool that you can import only what you actually need.

File: web/models/auth_handler.ex (hunk 2 above)
Comment (nburkley) [reply web/models/auth_handler.ex:17]: Version `1.3` of elixir provides a nicer way of doing this: http://tuvistavie.com/2016/elixir-1-3/#with-on-steroids

But for `1.2` this is the (ugly) syntax

File: priv/gettext/default.pot (hunk 1 above)
Comment (nburkley) [reply priv/gettext/default.pot]: ah, that's not even suppose to be there - thanks!

File: web/models/auth_handler.ex (hunk 2 above)
Comment (ghost) [reply web/models/auth_handler.ex:17]: Ah, alright, the new `else`. Cool. Good to know.

**Pull request merged**